### PR TITLE
Update chunked upload

### PR DESCRIPTION
**Pull request by HubertJaworski**

### CODE REVIEW

File: neptune/internal/storage/storage_utils.py [hunk 1]
```diff
@@ -190,7 +192,9 @@ def scan_unique_upload_entries(upload_entries):
     return walked_entries
 
 
-def split_upload_files(upload_entries, max_package_size=1 * 1024 * 1024, max_files=500):
+def split_upload_files(
+    upload_entries: Set[UploadEntry], max_package_size=1 * 1024 * 1024, max_files=500
```
Comment (pkasprzyk) [inline neptune/internal/storage/storage_utils.py:196]: `max_package_size=BYTES_IN_ONE_MB`?

**Review by pkasprzyk — CHANGES_REQUESTED** [review]

File: neptune/internal/hardware/constants.py [hunk 2]
```diff
@@ -15,4 +15,5 @@
 #
 
 BYTES_IN_ONE_MB = 2 ** 20
+BYTES_IN_FIVE_MB = 5 * 2 ** 20
```
[the remark this answers is not in the document]
Comment (HubertJaworski) [reply neptune/internal/hardware/constants.py:18]: actually, this reminds me - we should ask server for upload size, not have it as a constant (wtih a fallback to 5 mb)

File: neptune/internal/storage/datastream.py [hunk 3]
```diff
@@ -39,11 +40,15 @@ def __eq__(self, other):
...
+    fobj: Union[BinaryIO, io.BytesIO]
+    filename: str
+    length: int
+
+    def __init__(self, upload_entry: UploadEntry):
         self.filename = upload_entry.target_path
         if upload_entry.is_stream():
             self.fobj = upload_entry.source_path
-            self.length = None
+            self.length = self.fobj.getbuffer().nbytes
```
[the remark this answers is not in the document]
Comment (HubertJaworski) [reply neptune/internal/storage/datastream.py:51]: Please verify the possible types of objects stored in UploadEntry - its either a BytesIO or a file, and only BytesIO responses with is_stream() == True

File: neptune/internal/storage/storage_utils.py (hunk 1 above)
Comment (HubertJaworski) [reply neptune/internal/storage/storage_utils.py:196]: thank you

File: neptune/internal/storage/datastream.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (pkasprzyk) [reply neptune/internal/storage/datastream.py:51]: I have :-] https://github.com/neptune-ai/neptune-client/blob/master/neptune/internal/storage/storage_utils.py#L184 -- source path is `os.path.join(root, filename)`

[the remark this answers is not in the document]
Comment (pkasprzyk) [reply neptune/internal/storage/datastream.py:51]: Ok, I read though the flow and I hate my life, but it works.

**Review by pkasprzyk — APPROVED** [review]

**Pull request merged**